docs/guides/edgemesh_test_env_guide.md

```diff
@@ -1,23 +1,14 @@
-# Prerequisites
-- install containerd
-- ensure that have add follow config items to section ```edged``` in edge.yaml:
-```
-edged:
-    runtime-type: remote
-    remote-runtime-endpoint: /var/run/containerd/containerd.sock
-    remote-image-endpoint: /var/run/containerd/containerd.sock
-    runtime-request-timeout: 2
-    podsandbox-image: k8s.gcr.io/pause
-```
-- install CNI plugin
-
 # Edgemesh test env config guide
+## Docker support (recommended)
+* Refer to the [Usage](./../setup/installer_setup.md) to prepare the kubeedge environment. 
+* Then follow the [EdgeMesh guide](#Edgemesh-end-to-end-test-guide) to deploy Service
 
-## Install Containerd 
-+ please refer to the following link to install  
-> <https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd>
+## Containerd Support
+* Refer to the [Usage](./../setup/installer_setup.md) to prepare the kubeedge environment. 
+* Following steps must be taken for the container network configuration if choose containerd as the container engine. 
+> Note: CNI plugin installation and port mapping configuration only needed for containerd
```

Comment:
@sids-b I wrote a note here, is it feasible?

Reply:
Thanks @arcanique